frontend/src/state/actions/studentActions.ts

```diff
@@ -6,80 +6,83 @@ import {
 } from "../../../../common/types";
 import { IUserData } from "../../models/types";
 
-export const setUserAction = createAction(
-  "user/SET_USER",
-  (user: IUserData) => ({
-    user,
+export const setStudentAction = createAction(
+  "student/SET_USER",
+  (student: IUserData) => ({
+    student,
   })
 )();
 
 export const setAcademicYearAction = createAction(
-  "user/SET_ACADEMIC_YEAR",
+  "student/SET_ACADEMIC_YEAR",
   (academicYear: number) => ({
     academicYear,
   })
 )();
 
 export const setGraduationYearAction = createAction(
-  "user/SET_GRADUATION_YEAR",
+  "student/SET_GRADUATION_YEAR",
   (graduationYear: number) => ({
     graduationYear,
   })
 )();
 
 export const setUserMajorAction = createAction(
```

Comment:
this and a couple other actions should probably be changed to setStudent_XXX_ as well